packages/main/src/DateComponentBase.ts

```diff
@@ -162,6 +188,14 @@ class DateComponentBase extends UI5Element {
 			});
 	}
 
+	getISOFormat() {
```

Comment:
since this is a fallback instance we can store that instance somewhere instead of taking a new one on each iteration.